src/transformers/models/layoutlmv2/processing_layoutlmv2.py

```diff
@@ -122,6 +125,21 @@ def __call__(
         )
 
         # add pixel values
-        encoded_inputs["image"] = features.pop("pixel_values")
+        images = features.pop("pixel_values")
+        if return_overflowing_tokens is True:
+            images = self.get_overflowing_images(images, encoded_inputs["overflow_to_sample_mapping"])
+        encoded_inputs["image"] = images
 
         return encoded_inputs
+
+    def get_overflowing_images(self, images, overflow_to_sample_mapping):
+        # in case there's an overflow, ensure each `input_ids` sample is mapped to its corresponding image
+        images_with_overflow = []
+        for sample_idx in overflow_to_sample_mapping:
+            images_with_overflow.append(images[sample_idx])
+
+        assert len(images_with_overflow) == len(
+            overflow_to_sample_mapping
+        ), f"Expected length of images to be the same as the length of overflow_to_sample_mapping, but got {len(images_with_overflow)} and {len(overflow_to_sample_mapping)}"
```

Comment:
No new assert in the code base, please use a test and raise a `ValueError` here.

Reply:
Just added a test and changed the `assert` to `ValueError`. The reason for no `assert` is because `assert` is mostly used only for debugging purpose (detecting programmer error), which can get muted during production, right?